composer.json

```diff
@@ -63,6 +63,7 @@
     },
     "config": {
         "sort-packages": true,
+        "preferred-install": "source",
```

Comment:
@voidtek why do we need to set source as preferred method for installation?

Reply:
I will revert this change.